scripts/docs-generation/protocolDocs.ts

```diff
@@ -78,6 +78,10 @@ export function generateProtocolDocs (sidebars: any) {
                     description.description = protocolNote
                 }
 
+                if (description.deprecated) {
+                    description.deprecated = `This protocol command is deprecated <br /><br />${description.deprecated}`
```

Comment:
One seems fine to me
```suggestion
                    description.deprecated = `This protocol command is deprecated<br />${description.deprecated}`
```